erizo_controller/erizoClient/src/Stream.js

```diff
@@ -68,6 +77,77 @@ const Stream = (altConnectionHelpers, specInput) => {
     that.local = true;
   }
 
+  const setMaxVideoBW = (maxVideoBW) => {
+    if (that.local) {
+      const translated = (maxVideoBW * 1000 * 0.90) - (50 * 40 * 8);
+      log.info(`message: Setting maxVideoBW, streamId: ${that.getID()}, maxVideoBW: ${maxVideoBW}, translated: ${translated}`);
+      that.maxVideoBW = translated;
+    } else {
+      that.maxVideoBW = maxVideoBW;
+    }
```

Comment:
should we reapply the encoder parameters here?

Reply:
We don't always want to apply it immediately. For instance, if you specify a maxVideoBW in `room.publish` we then generate the initial configuration and then include those in the transceiver.